docs/DailyBuilds.md

```diff
@@ -14,6 +14,10 @@ If you want to download the latest daily build and use it in a project, then you
       <packageSources>
           <clear />
           <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
+          <add key="extensions" value="https://dotnetfeed.blob.core.windows.net/aspnet-extensions/index.json" />
+          <add key="entityframeworkcore" value="https://dotnetfeed.blob.core.windows.net/aspnet-entityframeworkcore/index.json" />
+          <add key="aspnetcore-tooling" value="https://dotnetfeed.blob.core.windows.net/aspnet-aspnetcore-tooling/index.json;" />
```

Comment:
Does the semicolon at the end cause issue for the NuGet client? (applies to the next line, too)

Reply:
Good catch. I'm not sure about what problems it might cause but I'll clean these up in a moment&hellip;